packages/create-neutrino-project/commands/init/index.js

```diff
@@ -0,0 +1,187 @@
+const { ensureDirSync, readJsonSync, writeJsonSync } = require('fs-extra');
+const path = require('path');
+const chalk = require('chalk');
+const stringify = require('javascript-stringify');
+const Generator = require('yeoman-generator');
+const questions = require('./questions');
+const {
+  PROJECTS,
+  LIBRARIES,
+  LINTERS,
+  isYarn,
+  isObjectEmpty
+} = require('./utils');
+
+module.exports = class Project extends Generator {
+  _makeRcFile(data) {
+    const config = { use: [] };
+
+    config.use.push(data.projectType);
+
+    switch (data.linter) {
+      case LINTERS.AIRBNB:
+        config.use.push(
+          data.projectType === PROJECTS.REACT ? LIBRARIES.NEUTRINO_PRESET_AIRBNB : LIBRARIES.NEUTRINO_PRESET_AIRBNB_BASE
+        );
+        break;
+      case LINTERS.STANDARDJS:
+        config.use.push(LIBRARIES.NEUTRINO_MIDDLEWARE_STANDARDJS);
+        break;
+      default:
+        break;
+    }
+
+    if (data.testRunner) {
+      config.use.push(data.testRunner);
+    }
+
+    return `module.exports = ${stringify(config, null, 2)};\n`;
+  }
+
+  _getDependencies() {
+    const devDependencies = [];
+    const dependencies = [];
+
+    switch (this.data.projectType) {
+      case PROJECTS.REACT:
+        dependencies.push(LIBRARIES.REACT, LIBRARIES.REACT_DOM, LIBRARIES.REACT_HOT_LOADER);
+        devDependencies.push(LIBRARIES.NEUTRINO_PRESET_REACT);
```

Comment:
I think all of these for React should be the same (either `dependencies` or `devDependencies`). Once built, the react packages are redundant. It's worth noting that Heroku only installs `dependencies` rather than `devDependencies` by default, so it really depends on the user's setup here.

Reply:
Agreed. I try to make the distinction about app dependencies vs. project/build/dev dependencies. What do you import in the app vs. what do you need to build the app?

Reply:
Going off on a bit of a tangent here - but IMO the real problem is that there are three buckets of dependencies (run-time, build-time, dev/testing) and only two places to put them (`dependencies`, `devDependencies`). And even 'build-time' is a bit nebulous, since that differs for people who commit the built assets to source control vs those who generate at deploy time.

Reply:
Good point! Too bad you cant have user defined dependencies:

```json
{
  "dependencies": {
    "default": {
      "react": "*"
    },
    "build": {
      "neutrino": "*"
    },
    "test": {
      "neutrino-preset-react": "*"
    }
  }
}
```

Anyway, probably a pipe dream, but would be neat. :smile:

Reply:
This still needs updating

Reply:
If we move neutrino deps to normal deps, it would go against our installation process:

https://www.npmjs.com/package/neutrino-preset-react#installation

Reply:
Most projects out there still split their deps into app code vs. non-app code.

Reply:
(Like Node.js projects)

Reply:
I would argue that those are wrong. `neutrino` and `neutrino-preset-react` are only ever needed at the same time as react-dom, there isn't a scenario when one is needed and the other not. For `neutrino-preset-airbnb-base`, that does belong in `devDependencies` on the other hand.

Though given that this PR is consistent with the docs, it seems fine to ignore for now, and potentially discuss alternatives in the future :-)

Reply:
Yes, from a logical standpoint, you are correct. These are idioms that are borrowed from Node.js, but aren't quite congruent to the nature of building a front-end project, since the dependencies get bundled at build time, unlike building Node.js projects. We try to give *the appearance* of similarity by keeping app dependencies separate from non-app dependencies, but it's merely a facade. It's some semblance of consistency between our projects.

Reply:
IMO, `neutrino-preset-react` is a development dependency, since it is only needed at build time. Although the preset itself could have `react-dom` as a peer dependency, indicating the user he/she should install that as well. That's what I do in https://github.com/barraponto/neutrino-preset-vue/blob/master/package.json

Reply:
(then, when the user installs it, it should go into actual dependencies)